doc/release/migrating_from_legacy_connect_framework.rst

```diff
@@ -0,0 +1,30 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _release/migrating_from_legacy_connect_framework:
+
+************************************
+Migrating from old Connect framework
```

Comment:
@henriknorin-ftrack 

old code was under the name:

**ftrack connectors** , users might not know it as **legacy framework** .